cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,15 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
-    
+    cy.visit('https://react-redux.realworld.io');
+
+    cy.get(':nth-child(2) > .nav-link').click();
+
+    cy.get(':nth-child(1) > .form-control').type('boqasuru@mailinator.com')
+    cy.get(':nth-child(2) > .form-control').type('My123Password!')
+
+    cy.get('.btn').click();
+
+    cy.get(':nth-child(4) > .nav-link');
```

Comment:
please, add assertion that there is username in this element, as it is required in the task description. you can use method `should()` for it